pytorch_lightning/loggers/wandb.py

```diff
@@ -217,6 +371,56 @@ def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None) ->
         else:
             self.experiment.log(metrics)
 
+    @rank_zero_only
+    def log_table(
+        self,
+        key: str,
+        columns: List[str] = None,
+        data: List[List[Any]] = None,
+        dataframe: Any = None,
+        step: Optional[int] = None,
+    ) -> None:
+        """Log a Table containing any object type (text, image, audio, video, molecule, html, etc).
+
+        Can be defined either with `columns` and `data` or with `dataframe`.
+        """
+
+        metrics = {key: wandb.Table(columns=columns, data=data, dataframe=dataframe)}
+        self.log_metrics(metrics, step)
+
+    @rank_zero_only
+    def log_text(
+        self,
+        key: str,
+        columns: List[str] = None,
+        data: List[List[str]] = None,
+        dataframe: Any = None,
+        step: Optional[int] = None,
+    ) -> None:
+        """Log text as a Table.
+
+        Can be defined either with `columns` and `data` or with `dataframe`.
+        """
+
+        self.log_table(key, columns, data, dataframe, step)
+
+    @rank_zero_only
+    def log_image(self, key: str, images: List[Any], **kwargs: str) -> None:
+        """Log images (tensors, numpy arrays, PIL Images or file paths).
+
+        Optional kwargs are lists passed to each image (ex: caption, masks, boxes).
+        """
+        if not isinstance(images, list):
+            raise TypeError(f'Expected a list as "images", found {type(images)}')
```

Comment:
would we be ok here if we just converted it to a list of a single element?

Reply:
Sure, should I add a warning as well?